dnx-on-ubuntu/azuredeploy.parameters.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachines_dnx_on_ubuntu_adminPassword": {
```

Comment:
See https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#parameters-file-placeholders for appropriate parameters